content/blog/2024-12-20-2024-year-of-valkey/index.md

```diff
@@ -0,0 +1,79 @@
+
++++
+title=  "2024: The Year of Valkey"
+description = "The end of the calendar year is a great time to reflect, but for Valkey this particular year-end holds special meaning."
+date= 2024-12-20 01:01:01
+authors= ["kyledvs"]
++++
+
+The end of the calendar year is a great time to reflect, but for Valkey this particular year-end holds special meaning.
+Think about it: this time in 2023, no one had ever heard the name “Valkey” because, well, it didn’t exist.
+This seems nearly unbelievable given how much has changed in only a few short months.
+
+Now, at the end of 2024, Valkey has had both a minor and major release as well as a few patches.
+Valkey 7.2 primarily introduce the project and new name while carrying over the feature set and performance from before the fork.
+Valkey 8.0 made substantial internal changes, bringing higher performance through multi-threaded I/O, better memory efficiency from a rewritten main dictionary, and more granular visibility into performance and resource usage.
+In 2025, the project is looking toward a future with new functionality and a whole boatload of optimizations in both performance and efficiency.
+
+## Beginnings
+
+[![YouTube video thumbnail of open source summit north america](images/ossna-thumb.jpg)](https://youtu.be/74Svvu37I_8?si=onLIvlu3X_ncKdh2&t=3055)
+
+Getting started with Valkey has changed substantially over the course of the year.
+When Valkey first launched at Open Source Summit North America, I recall excitedly telling people that you could build from source, get the binary from the website or even use a container.
+Now, you can get Valkey directly from the [package manager in most Linux distributions](https://repology.org/project/valkey/versions) (and more on the way in 2025).
+There are multiple options for containerization and operators to fit your needs.
+And, for those who want to let *others* run Valkey, it is also available as a service on Aiven, AWS, Google Cloud Platform, NetApp Instaclustr, UpCloud, and several more.
```

Comment:
```suggestion
And, for those who want to let *others* run Valkey, it is also available as a service on Aiven, AWS, Google Cloud Platform, NetApp Instaclustr, UpCloud, and several more.
```
Is that the preferred way to write out GCP? Just odd that it's not written out like AWS.

Reply:
I think this is also a fully managed Valkey, https://elest.io/open-source/valkey.

Reply:
Okay! I believe they prefer 'Google Cloud Platform' but for symmetry I'll do full name + parenthetical e.g. `Amazon Web Services (AWS)` and `Google Cloud Platform (GCP)`

The listed companies are ones have had formal interactions with the project. I don't believe that elastio has.